package.json

```diff
@@ -189,32 +189,33 @@
     "build:prod": "webpack --mode production",
     "build:test": "tsc",
     "pretest": "npm run build:prod && npm run build:test",
-    "update-pmd": "cd pmd-packager && ./upgrade.sh",
+    "update-pmd-package": "cd pmd-packager && ./upgrade.sh",
+    "update:pmd:latest": "./bin/latestversion.sh && npm run update:pmd:specified",
+    "update:pmd:specified": "./bin/update.sh",
```

Comment:
If the pmd-version.txt file has a different version, pull that specific version.
This is where we can hook in user overrides in extension settings